src/enrich/derive/metrics/get-stability-metrics.js

```diff
@@ -38,14 +38,6 @@ function upsertFolderAttributes(pAllMetrics, pModule, pDirname) {
   return pAllMetrics;
 }
 
-function orderFolderMetrics(pLeftMetric, pRightMetric) {
```

Comment:
removed because sorting is a responsibility for a reporter module (which will do this anyway)